docs/content/users/install/ddev-installation.md

```diff
@@ -250,8 +270,6 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
     * We recommend using [Chocolatey](https://chocolatey.org/). Once installed, you can run `choco install ddev docker-desktop git` from an administrative shell. You can upgrade by running `ddev poweroff && choco upgrade ddev`.
     * Each [DDEV release](https://github.com/ddev/ddev/releases) includes a Windows installer (`ddev_windows_installer.<version>.exe`). After running that, you can open a new Git Bash, PowerShell, or cmd.exe window and start using DDEV.
 
-    Most traditional Windows users will want to enable Mutagen for superb performance; no installation is required; run `ddev config global --mutagen-enabled`. It still won't be as fast as one of the WSL2 options.
```

Comment:
I removed it because it’s redundant; there’s already a note [leading these instructions](https://github.com/ddev/ddev/blob/7e5643e81646becf20bbf3434bf848d89f224898/docs/content/users/install/ddev-installation.md?plain=1#L268).